core/src/main/scala/com/typesafe/tools/mima/core/ClassInfo.scala

```diff
@@ -72,18 +75,21 @@ private[mima] sealed abstract class ClassInfo(val owner: PackageInfo) extends In
   final def fields: Members[FieldInfo]   = afterLoading(_fields)
   final def methods: Members[MethodInfo] = afterLoading(_methods)
   final def flags: Int                   = afterLoading(_flags)
+  final def isPackagePrivate: Boolean    = afterLoading(_packagePrivate)
```

Comment:
Rename to `scopedPrivate` (or similar), as `private[C]` means private to an enclosing `C` scope, which needn't be a package.